omod/src/main/webapp/admin/users/userForm.jsp

```diff
@@ -198,7 +198,13 @@
 				</tr>
 			</c:if>
 			<tr><td colspan="2">&nbsp;</td></tr>
-			
+			<tr>
+				<td colspan="2">&nbsp;
+					<openmrs:extensionPoint pointId="org.openmrs.locations" type="html">
```

Comment:
@PremnishaTW so this extension point will always be limited to be used with 'locations'?

Reply:
@mks-d The intent is to use it for locations, but it depends on the implementation of the extension point